.github/workflows/top-issues.yml

```diff
@@ -0,0 +1,27 @@
+name: Top Issues Action
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '12 05 */1 * *'
+
+jobs:
+  ShowTopIssues:
+    name: Display top issues
+    runs-on: ubuntu-latest
+    steps:
+      - name: Run top issues action
+        uses: rickstaa/top-issues-action@v1
+        env:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Shouldn't this be in the `with:` group?

Reply:
I can't answer that. I was using the [provided example](https://github.com/marketplace/actions/top-issues-action#outputs) as a template.